scripts/sql/delta/v1_9_0-v1_10_0/apolloportaldb-v1_9_0-v1_10_0-after.sql

```diff
@@ -0,0 +1,78 @@
+--
+-- Copyright 2021 Apollo Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+# delta schema to upgrade apollo portal db from v1.9.0 to v1.10.0
+
+Use ApolloPortalDB;
+
+UPDATE `App` SET `DeletedAt` = -Id WHERE `IsDeleted` = 1 and `DeletedAt` = 0;
```

Comment:
Is it possible to keep the original `DataChange_LastTime` value? I found it useful to keep the original time when Diagnosing the problems.

Reply:
I think the previous comment is still valid, we may keep the original `DataChange_LastTime value` via SQL like below?

```suggestion
UPDATE `App` SET `DeletedAt` = -Id, `DataChange_LastTime` = `DataChange_LastTime` WHERE `IsDeleted` = 1 and `DeletedAt` = 0;
```

Reply:
Good to me, already updated.